### PR TITLE
Corrected query pattern for requests & Added time zone offset and limit field to the request

### PR DESCRIPTION
**Pull request by Achint08**
- Added time zone offset and limit field to the request

**Changes proposed in this pull request**
- Space needs to be replaced by '+' in query field for request
- If  there is  '#' in the query field , query pattern needs to be like :- `%23fossasia#fossasia` for more results. Added this.
- Added timezoneOffset and limit field to the request.
- Added field to get minified reponse for `user.json` request.

**Link to live demo (if appropriate):** https://achint08.github.io/loklak_search/

### CODE REVIEW

**Comment by Achint08** [comment]
@daminisatya Not as such. These changes are introduced due to new pattern for search queries. 
I researched about what should be pattern for queries to get more and faster results :-
1. If we search for `#fossasia` at [loklak search](http://loklak.net/search?query=%23fossasia) , results strength is less compared to this new introduced change [here](https://achint08.github.io/loklak_search/) . This goes for all queries with hashtags.
2. There was a change introduced that blank space needs to be replaced by `+` in the query as discussed aboved.
3. Previously, time zone offset field was not added. I introduced this change because this would save time from processing timestamp of the tweets. Limit field is added (and set to 10) because in the sidebar , only top 10 mentions/hashtags/twitters are displayed.

File: src/app/effects/api-search.effects.ts [hunk 1]
```diff
@@ -42,14 +42,18 @@ export class ApiSearchEffects {
 					.map((action: apiAction.SearchAction) => action.payload)
 					.switchMap(query => {
 						
-						let URIquery = encodeURIComponent(query.queryString);
+						let normalizedquery = query.queryString.replace(/ /g, '+');
+						let encodedquery = encodeURIComponent(query.queryString);
+						if(normalizedquery[0] === '#') {
```
Comment (hemantjadon) [inline src/app/effects/api-search.effects.ts]: What if the query is of type `fossasia #loklak`?
Then i guess this simple comparision with first character will fail. What are your views?
Should normalization be the part of an effect?

Comment (Achint08) [inline src/app/effects/api-search.effects.ts]: I will correct the above thing. And where should be normalization done then? :)

Comment (Achint08) [inline src/app/effects/api-search.effects.ts]: Checkout now @hemantjadon .

**Comment by mariobehling** [comment]
@hemantjadon Thanks! Please follow best practices more tightly in following commits and provide an issue for any PR beforehand.

**Pull request merged**